src/ontology/mondo-edit.obo

```diff
@@ -303765,14 +303765,14 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_disease {source="Orphanet:314647"}
 subset: orphanet_rare {source="Orphanet:314647"}
 subset: rare
-synonym: "CANPMR" RELATED DEPRECATED [MONDO:Lexical, OMIM:614756]
-synonym: "cerebellar ataxia, nonprogressive, with intellectual disability" RELATED [MONDO:Lexical, OMIM:614756]
-synonym: "cerebellar ataxia, nonprogressive, with mental retardation" RELATED DEPRECATED [MONDO:Lexical, OMIM:614756]
+synonym: "CAMTA1-related disorder" EXACT [https://orcid.org/0000-0001-9310-0163, PMID:38044714]
+synonym: "CANPMR" EXACT DEPRECATED [MONDO:Lexical, OMIM:614756]
+synonym: "cerebellar ataxia, nonprogressive, with intellectual disability" EXACT [MONDO:Lexical, OMIM:614756]
+synonym: "cerebellar ataxia, nonprogressive, with mental retardation" EXACT DEPRECATED [MONDO:Lexical, OMIM:614756]
 synonym: "cerebellar dysfunction with variable cognitive and behavioral abnormalities" EXACT [OMIM:614756, OMIM:genemap2]
-synonym: "non-progressive cerebellar ataxia with intellectual disability" RELATED [Orphanet:314647]
-synonym: "nonprogressive cerebellar ataxia with intellectual disability" RELATED [DOID:0050998]
-synonym: "nonprogressive cerebellar ataxia with mental retardation" RELATED DEPRECATED [DOID:0050998]
-synonym: "nonprogressive cerebellar atxia with intellectual disability" EXACT CLINGEN_LABEL []
+synonym: "non-progressive cerebellar ataxia with intellectual disability" EXACT [Orphanet:314647]
+synonym: "nonprogressive cerebellar ataxia with intellectual disability" EXACT CLINGEN_LABEL [DOID:0050998, Orphanet:314647]
```

Comment:
I don't see that Orphanet:314647 lists "nonprogressive cerebellar ataxia with intellectual disability" only the version with the hyphen.

Reply:
done